### PR TITLE
Add blog post on Java Developer

### PR DESCRIPTION
**Pull request by BirgitBader**
@jensfischerhh please review

### CODE REVIEW

File: _posts/2016/2016-03-01-hr-java.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+My name is Jens.
+I'm 40 years old and I'm working as a Senior Java Engineer at ePages since February 2015.
+
+How did it happen that I'm working at ePages?
+Well, I heard that Harm Behrens started a new, exciting job as CTO at ePages.
+I knew him already for quite a while and when talking to him, I found out more about ePages as a potential employer.
+We talked about the upcoming projects and plans and I then decided that I'd like the idea to accompany Harm with these topics.
+
+But I also wanted to get a personal impression about the company and the development processes.
+So I dropped by one afternoon to sit with some developers who showed me how they code.
```
Comment (jensfischerhh) [inline _posts/2016/2016-03-01-hr-java.md]: So one afternoon I dropped by to sit...

File: _posts/2016/2016-03-01-hr-java.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+ePages offers a very informative introduction week.
+I was intensively coached about the product and got development-specific insights.
+But it was also about non-development related stuff like ePages academy, plans of product management, scrum values etc.
+Besides that, I got to know other ePages newbies there.
+
+### And your team?
+
+Well, besides the training modules, I had short slots with the team.
+I was shown, what they're currently working on, how the sprint is going - and of course we had lunch together.
```
Comment (jensfischerhh) [inline _posts/2016/2016-03-01-hr-java.md]: I was shown what they're currently working on, ...

File: _posts/2016/2016-03-01-hr-java.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+To being able to completely focus on the new architecture, we had to decouple this topic from the other tasks in the team.
+We split the team into two.
+Team Black stepped deep into the new architecture whereas the other colleagues focus on developing our REST API.
+
+### What does a typical work day look like?
+
+{% image blog/blog-team-black-2.jpg %}{% endimage %}
+
+I'm usually the early-bird in the team: I start between 8:00 and 8:30.
```
Comment (jensfischerhh) [inline _posts/2016/2016-03-01-hr-java.md]: one of the early-birds

File: _posts/2016/2016-03-01-hr-java.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+Then I work myself through some smaller tasks, organisational stuff or do some bug fixing until we have our Daily Standup at 9:45.
+After having discussed the tasks for the day, we start single or pair programming sessions or discuss in detail about architectural concepts.
+
+Between 12:00 and 13:00 we all have lunch together.
+It's good to go somewhere outside, to have the head cleared.
+
+The afternoon is similar to the morning.
+Often we do sessions on the projector then, to discuss coding results with the whole team.
```
Comment (jensfischerhh) [inline _posts/2016/2016-03-01-hr-java.md]: Often we do team-wide code review session using the projector, to discuss...

File: _posts/2016/2016-03-01-hr-java.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+### And your team?
+
+Well, besides the training modules, I had short slots with the team.
+I was shown, what they're currently working on, how the sprint is going - and of course we had lunch together.
+
+### What happened after that week?
+
+I just jumped into the cold water.
+Originally, I was hired to start developing a new software architecture together with other colleagues.
+But first I supported the development of the ePages App & Theme Store.
```
Comment (jensfischerhh) [inline _posts/2016/2016-03-01-hr-java.md]: App & Themes Store?

File: _posts/2016/2016-03-01-hr-java.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+I also visited the Jena office to build a bridge to the colleagues I would be closely working with in the future.
```
Comment (jensfischerhh) [inline _posts/2016/2016-03-01-hr-java.md]: to build a connection to

File: _posts/2016/2016-03-01-hr-java.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+Communication is way easier, when you've already spent some time together.
+
+### The App & Theme Store is launched. What are you focussing on now?
```
Comment (jensfischerhh) [inline _posts/2016/2016-03-01-hr-java.md]: App & Themes Store?

File: _posts/2016/2016-03-01-hr-java.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+### What does a typical work day look like?
+
+{% image blog/blog-team-black-2.jpg %}{% endimage %}
+
+I'm usually the early-bird in the team: I start between 8:00 and 8:30.
+Then I work myself through some smaller tasks, organisational stuff or do some bug fixing until we have our Daily Standup at 9:45.
+After having discussed the tasks for the day, we start single or pair programming sessions or discuss in detail about architectural concepts.
+
+Between 12:00 and 13:00 we all have lunch together.
```
Comment (jensfischerhh) [inline _posts/2016/2016-03-01-hr-java.md]: Between around 12:00 and 13:00

File: _posts/2016/2016-03-01-hr-java.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+
+### Which technologies do you use?
+
+We mainly work with Spring Boot, Docker, Kubernetes, RabbitMQ and MySQL.
+
+### What do you like at epages?
+
+The colleagues are all very open-minded and there's a friendly family atmosphere in the whole company.
+Between all teams there's an open interest for knowledge transfer.
+We also stepped deeper into the developer community and open more to the external development world.
```
Comment (jensfischerhh) [inline _posts/2016/2016-03-01-hr-java.md]: We also stepped deeper into the local developer community by attending and hosting meetups and open more to the external development world.

File: _posts/2016/2016-03-01-hr-java.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+
+I also like the cool company events, like the YOU or internal hackathons.
+
+### What qualities does an applicant need to fit into the team?
+
+Those who want to join Team Black should have children (*Jens is grinning*).
+Just kidding.
+But honestly, our team is above average with regards to children :wink:.
+
+They should not be afraid of criticism.
```
Comment (jensfischerhh) [inline _posts/2016/2016-03-01-hr-java.md]: constructive criticism and feedback.

File: _posts/2016/2016-03-01-hr-java.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+We have intensive and professional discussions in the team, so they should be tough and convinced of their opinion.
```
Comment (jensfischerhh) [inline _posts/2016/2016-03-01-hr-java.md]: We often have intensive but professional discussion in the team, so they should be communicative and convinced of their opinion.

**Comment by jensfischerhh** [comment]
Review done, see all inline comments!

File: _posts/2016/2016-03-01-hr-java.md (hunk 5 above)
Comment (BirgitBader) [inline _posts/2016/2016-03-01-hr-java.md]: According to PM it's App & Theme Store

**Comment by BirgitBader** [comment]
@jensfischerhh I added all remarks and also the ones from Serena. Please give me a short hint here, if you agree with the post. Then I will publish it on March, 1st.

**Comment by jensfischerhh** [comment]
Monty Nygard would call it: "Release it!"

**Pull request merged**